pkg/jobrunaggregator/jobrunaggregatorapi/types_row_historical_data.go

```diff
@@ -44,7 +47,7 @@ func (a *AlertHistoricalDataRow) GetJobData() HistoricalJobData {
 	return a.HistoricalJobData
 }
 func (a *AlertHistoricalDataRow) GetName() string {
-	return fmt.Sprintf("%s %s %s", a.AlertName, a.AlertNamespace, a.AlertLevel)
+	return a.AlertName
```

Comment:
Seems like we are loosing context here.  So if an alert with the same name but different namespace exists we can no longer tell the difference based on the name.  Looks like this is only used in a markdown table currently did you really need this change?

Reply:
reverted back to original